README.md

```diff
@@ -187,6 +242,12 @@ credentials in the application's environment, and might look something like
 `host/cf/0299a19d-7de4-4e98-89f6-372ac7c0521f` (for example, if your `CONJUR_POLICY`
 was set to `cf`).
```

Comment:
What do you think about slightly changing the example of `cf` as the value of CONJUR_POLICY? If a company has multiple foundations, then `cf` is not a scalable name. Maybe we should recommend creating a policy called `cf` under `root` and then for each foundation, have its own policy like `production` then suggest that `CONJUR_POLICY` would have the value `cf/production`.
What do you think?

Reply:
I think this is a great improvement! I will make the change.